csun-metalab/nomi
# NAM-555 -- UX: Icon for upload modal is placed differently relative to the image on the profile page and the gallery page. We should conform to the Gallery Page

# Description
Re-positioned edit button to be similar to `student profile` page.
  
# Testing
1. Login as `steve` .
2. Visit the `student gallery` page and ensure the edit button position is similar to any `student profile` edit button.
